api-report/container-runtime.api.md

```diff
@@ -332,6 +336,7 @@ export interface IConnectableRuntime {
 
 // @public
 export interface IContainerRuntimeOptions {
+    readonly chunkSizeInBytes?: number;
```

Comment:
Not sure if name tells anything to uneducated developer. Maybe maxOpSize?

Reply:
I.e. focus on observable behaviors, not on implementation details of how we achieve it in giving name?

Reply:
Not sure if I agree with this, especially when talking about observable behaviors. MaxBatchSize is a threshold. MaxOpSize also sounds like a threshold, which begs the question: 'what happens if the max op size is exceeded?'. If the maxBatchSize is exceeded (without compression) we fail to send the payload, so the customer should make smaller payloads. In the meantime, exceeding MaxOpSize has no effect wrt to the immediate success factor of said payload. It's just the value we use as a denominator in the event of having to chunk ops. This value can be used to soften the blow wrt to the volume of ops sent, but it is ultimately not a threshold.

Reply:
I'd at least add "Max" to the name